internal/session/handle_expunge.go

```diff
@@ -29,7 +29,7 @@ func (s *Session) handleExpunge(ctx context.Context, tag string, cmd *proto.Expu
 	return response.Ok(tag).WithMessage("EXPUNGE"), nil
 }
 
-func (s *Session) handleUIDExpunge(ctx context.Context, tag string, cmd *proto.UIDExpunge, mailbox *state.Mailbox, ch chan response.Response) (response.Response, error) {
+func (s *Session) handleUIDExpunge(ctx context.Context, tag string, cmd *command.UIDExpungeCommand, mailbox *state.Mailbox, ch chan response.Response) (response.Response, error) {
```

Comment:
Didn't we want to remove the Command suffix on all command types? Maybe you forgot this one

Reply:
Done